veloroutes_voies_vertes/actions.py

```diff
@@ -202,3 +204,95 @@ def update_selected_feature(*args):
             _add_selected_from_relation(agg_id, 'id_portion', 'segment', 'id_segment', 'element')
 
     layers[0].triggerRepaint()
+
+
+def create_relation(*args):
+    layer_name = args[0]
+    layer_update = args[1]
+    rel_layer_name = args[2]
+    src_field_id_name = args[3]
+    rel_field_id_name = args[4]
+
+    layers = QgsProject.instance().mapLayersByName(layer_name)
+
+    # Control if layers exists
+    if not len(layers):
+        iface.messageBar().pushCritical('Véloroutes', 'La couche {} n\'a pas été trouvée'.format(
+            layer_name))
+        return
+
+    src_layer = layers[0]
+
+    layers = QgsProject.instance().mapLayersByName(layer_update)
+
+    if not len(layers):
+        iface.messageBar().pushCritical('Véloroutes', 'La couche {} n\'a pas été trouvée'.format(
+            layer_update))
+        return
+
+    update_layer = layers[0]
+
+    # count number of features selected from src_layer
+    count = src_layer.selectedFeatureCount()
+    if count < 1:
+        iface.messageBar().pushCritical(
+            'Véloroutes',
+            'Vous devez sélectionner au moins un objet de la couche {}'.format(layer_update))
+        return
+
+    fields = update_layer.fields()
+    new_feature = QgsFeature()
+    new_feature.setFields(fields)
+
+    features = src_layer.getSelectedFeatures()
+    selected_features_id = []
+    for feat in features:
+        selected_features_id.append(feat['id_portion'])
+
+    update_layer.startEditing()
+
+    # open feature form and test if cancel form for stop or not the script
+    if not iface.openFeatureForm(update_layer, new_feature, showModal=True):
+        update_layer.rollBack()
+        iface.messageBar().pushInfo(
+                    'Véloroutes',
```

Comment:
```suggestion
            'Véloroutes',
```
On garde le minimum d'indentation possible